add information when this feature is available (#557)
shopware/docs: guides/hosting/performance/performance-tweaks.md

[Read more](../infrastructure/elasticsearch/elasticsearch-setup.md)

## Prevent mail data updates
{% hint style="info" %}

> This feature will be available in Shopware 6.5 (and currently in trunk)

Reviewer: ```suggestion
This feature will be available starting with Shopware 6.5.
```
